boudicca.base/search-api/build.gradle.kts

```diff
@@ -2,10 +2,12 @@ plugins {
     kotlin("jvm")
     kotlin("kapt")
     id("io.cloudflight.autoconfigure.swagger-api-configure")
+    `maven-publish`
```

Comment:
those lines have nothing to do with the fix, do they? please remove them from the pr, otherwise it looks good, you can just merge them